CefSharp.Core/ManagedCefBrowserAdapter.h

```diff
@@ -661,9 +661,9 @@ namespace CefSharp
             }
         }
 
-        void RegisterJsObject(String^ name, Object^ object)
+		void RegisterJsObject(String^ name, Object^ object, bool lowerCaseJavascriptNames)
```

Comment:
Spaces not tabs see https://github.com/cefsharp/CefSharp/blob/master/CONTRIBUTING.md#coding-style